src/member/templates/client/view/notes.html

```diff
@@ -46,6 +46,7 @@ <h2 class="ui header">
         <div class="field">
             <a href="{{ request.path }}?display={{ display }}" class="ui button">{% trans "Reset" %}</a>
             <button class="ui yellow button" type="submit">{% trans "Search" %}</button>
+            <a href="{% url 'member:client_notes_add' client.pk %}" class="ui primary">{% trans "Add a Note" %}</a>
```

Comment:
Fixed in 479b430:
![image](https://cloud.githubusercontent.com/assets/8630726/20840993/2ea1f62c-b880-11e6-9c16-1081f7b97741.png)